### PR TITLE
FH-3624 - Change the cmd  appinit to follow the V3 standard and add u…

### PR DESCRIPTION
**Pull request by camilamacedo86**
JIRA: https://issues.jboss.org/browse/FH-3624

**Changes**
- Move to V3 Standard
- Add unit test

**Steps to verify**
- Create a new project
- deploy the cloud app
- go to the client app
- Take the id of the app and the app api key
- run the command `fhc appinit --app=<app-id> --key=<app-key>`

**Local Test**
```

Camilas-MacBook-Pro:fh-fhc cmacedo$ ./bin/fhc.js appinit xzauxfwgzpedqbcaxujffd6p 934ba74c28076931633bbc41507462c6e9d85894
{
  "apptitle": "Sync App",
  "domain": "support",
  "firstTime": false,
  "hosts": {
    "environment": "dev",
    "type": "cloud_nodejs",
    "url": "https://support-xzauxfuns4fnjafvu23kwhpx-dev.mbaas1.us.feedhenry.com"
  },
  "init": {
    "trackId": "p7rzdgjh65kv5mlvl53fy2dd"
  },
  "status": "ok"
}
Camilas-MacBook-Pro:fh-fhc cmacedo$ 

```

### CODE REVIEW

**Comment by wtrocki** [comment]
@camilamacedo86 - this is type of command that will be rarely used from cmd. It was 'probably' build for automation purposes. We need to make sure that this command will still work when importing fhc as library. Apart from that everything seems to be ok.

File: lib/cmd/fh3/appinit.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    'app' : i18n._("Unique 24 character GUID of your application."),
+    'key' : i18n._("App API Key of the client application"),
+    'params' : i18n._("Parameters for this init call")
+  },
+  'customCmd': function(argv, cb) {
+    var dataToSend = (argv.params) ? JSON.parse(argv.params) : {};
+    dataToSend.appid = argv.app;
+    dataToSend.appkey = argv.key;
+    dataToSend.destination = 'fhc';
+    dataToSend.cuid = '12345678790123456787901234567879012';
```
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply lib/cmd/fh3/appinit.js]: @wtrocki really tks. Yep. It worked perfectly fine without the strange fix info " dataToSend.cuid = '12345678790123456787901234567879012';"

[the remark this answers is not in the document]
Comment (camilamacedo86) [reply lib/cmd/fh3/appinit.js]: 👍 Done

**Comment by camilamacedo86** [comment]
@wtrocki the change requested was made and I looked and I could not find any automated test which is using it. Maybe in the future.

**Comment by wtrocki** [comment]
My platform just crashed so I cannot verify this but it seem to be ok.

**Comment by camilamacedo86** [comment]
@wtrocki Tks 👍

**Pull request merged**